mne/preprocessing/peak_finder.py

```diff
@@ -38,10 +37,13 @@ def peak_finder(x0, thresh=None, extrema=1, verbose=None):
 
     Example
```

Comment:
Should be called `Examples`

https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
But didn't they just release 1.0.0? How long until 1.1.0 will be out?

Reply:
It is already RC

https://mail.python.org/pipermail/scipy-dev/2018-April/022775.html